code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -1,4 +1,5 @@
 #define C2NAMEREAGENT	"[initial(reagent.name)] (Has Side-Effects)"
+#define CHEMEFFECT "[initial(reagent.name)][reagent_desc[reagent]] [initial(reagent.harmful) ? "(Has side effects)" : ""]"
```

Comment:
As it currently stands this doesn't check to see if the reagent is actually in reagent_desc, so it will runtime if we try and get a reagent that doesn't have an entry.

To check for an entry do something like this:
```suggestion
#define CHEMEFFECT "[initial(reagent.name)][(reagent in reagent_desc) ? reagent_desc[reagent] : ""] [initial(reagent.harmful) ? "(Has side effects)" : ""]"
```
Oh and you may want to add some spaces between the ][s so the textisn'tdisplayedlikethis.